Index: utils/fpcm/fpcmmain.pp
===================================================================
--- utils/fpcm/fpcmmain.pp.orig
+++ utils/fpcm/fpcmmain.pp
@@ -70,7 +70,7 @@ interface
         o_linux,o_go32v2,o_win32,o_os2,o_freebsd,o_beos,o_haiku,o_netbsd,
         o_amiga,o_atari, o_solaris, o_qnx, o_netware, o_openbsd,o_wdosx,
         o_palmos,o_macos,o_darwin,o_emx,o_watcom,o_morphos,o_netwlibc,
-        o_win64,o_wince,o_gba,o_nds,o_embedded,o_symbian
+        o_win64,o_wince,o_gba,o_nds,o_embedded,o_symbian,o_tyndur
       );
 
       TTargetSet=array[tcpu,tos] of boolean;
@@ -92,14 +92,14 @@ interface
         'linux','go32v2','win32','os2','freebsd','beos','haiku','netbsd',
         'amiga','atari','solaris', 'qnx', 'netware','openbsd','wdosx',
         'palmos','macos','darwin','emx','watcom','morphos','netwlibc',
-        'win64','wince','gba','nds','embedded','symbian'
+        'win64','wince','gba','nds','embedded','symbian','tyndur'
       );
 
       OSSuffix : array[TOS] of string=(
         '_linux','_go32v2','_win32','_os2','_freebsd','_beos','_haiku','_netbsd',
         '_amiga','_atari','_solaris', '_qnx', '_netware','_openbsd','_wdosx',
         '_palmos','_macos','_darwin','_emx','_watcom','_morphos','_netwlibc',
-        '_win64','_wince','_gba','_nds','_embedded','_symbian'
+        '_win64','_wince','_gba','_nds','_embedded','_symbian','_tyndur'
       );
 
       { This table is kept OS,Cpu because it is easier to maintain (PFV) }
@@ -132,7 +132,8 @@ interface
         { gba    }  ( false, false, false, false, false, true,  false, false, false),
         { nds    }  ( false, false, false, false, false, true,  false, false, false),
         { embedded }( true,  true,  true,  true,  true,  true,  true,  true,  true ),
-        { symbian } ( true,  false, false, false, false, true,  false, false, false)
+        { symbian } ( true,  false, false, false, false, true,  false, false, false),
+        { tyndur }  ( true,  false, false, false, false, false, false, false, false)
       );
 
     type
